src/confluent/report.md

```diff
@@ -0,0 +1,120 @@
+# Azure CLI Module Creation Report
```

Comment:
Is this file required?

Reply:
Generated and used by codegen, may be helpful for incremental codegen.